docs/developer-guide/syntax-reference.rst

```diff
@@ -1214,6 +1214,14 @@ Scalar functions
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
 | CONCAT                 |  ``CONCAT(col1, '_hello')``                                               | Concatenate two strings.                          |
 +------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| CURRENTDATE            |  ``CURRENTDATE()``                                                        | Gets an integer representing days since epoch.    |
++------------------------+---------------------------------------------------------------------------+---------------------------------------------------+
+| CURRENTTIMESTAMP       |  ``CURRENTTIMESTAMP([TIMEZONE])``                                         | Gets the current timestamp represented in         |
```

Comment:
"Gets the current timestamp in milliseconds, represented as a BIGINT."

Reply:
Done